app-backend/tile/src/main/scala/Router.scala

```diff
@@ -31,31 +31,29 @@ class Router extends LazyLogging
 
   def root = cors() {
     handleExceptions(tileExceptionHandler) {
-      pathPrefix("tiles") {
```

Comment:
You may already have this work queued up, but I think that the next step is to tweak the Nginx configuration for the tile service so that it rewrites `/tiles/...` to `/...`.

Reply:
I checked [this](https://github.com/raster-foundry/raster-foundry/blob/develop/nginx/etc/nginx/conf.d/tiler.conf) tiler Nginx config, and I only see a location block for `/`. From my understanding, the whole URI gets passed to the service and routing `/tiles` is handled by Scala. Is that correct?